app/renderer/components/frame/frame.js

```diff
@@ -682,15 +683,27 @@ class Frame extends React.Component {
         isSecure = false
         const parsedUrl = urlParse(this.props.location)
         ipc.send(messages.CHECK_CERT_ERROR_ACCEPTED, parsedUrl.host, this.props.tabId)
-      } else if (['warning', 'passive-mixed-content'].includes(e.securityState)) {
+      } else if (e.securityState === 'warning' ||
+                 e.securityInfo.mixedContentStatus === 'content-status-displayed') {
         // Passive mixed content should not upgrade an insecure connection to a
         // partially-secure connection. It can only downgrade a secure
         // connection.
         isSecure = this.props.isSecure !== false ? 1 : false
       }
+
+      if (e.securityInfo.securityLevel === 'ev-secure') {
+        if (e.securityInfo.certificate &&
+            e.securityInfo.certificate.organizationNames.length &&
+            e.securityInfo.certificate.countryName) {
```

Comment:
I think we shouldn't require countryName, since it's not a required field in EV certs according to https://cabforum.org/ev-certificate-contents/.

Reply:
addressed in 130d7ef77722cab1c3e416d6b74efdaf59005e6a